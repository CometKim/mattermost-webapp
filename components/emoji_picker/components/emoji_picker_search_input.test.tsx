// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import React from 'react';
import {mount} from 'enzyme';
import {IntlProvider} from 'react-intl';

import {defaultIntlConfig} from 'tests/helpers/intl-test-helper';

import EmojiPickerSearchInput from './emoji_picker_search_input';

describe('components/EmojiPicker/components/EmojiPickerSearchInput', () => {
    it('should match snapshot', () => {
        const wrapper = mount(
            <IntlProvider {...defaultIntlConfig}>
                <EmojiPickerSearchInput/>
            </IntlProvider>
        );
        expect(wrapper).toMatchSnapshot();
    });
});
