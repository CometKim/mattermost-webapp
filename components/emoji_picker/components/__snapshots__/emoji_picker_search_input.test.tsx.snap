// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/EmojiPicker/components/EmojiPickerSearchInput should match snapshot 1`] = `
<IntlProvider
  defaultFormats={Object {}}
  defaultLocale="en"
  formats={Object {}}
  locale="en"
  messages={Object {}}
  onError={[Function]}
  textComponent={Symbol(react.fragment)}
  timeZone="Etc/UTC"
>
  <ForwardRef(EmojiPickerSearchInput)>
    <div
      className="emoji-picker__search-container"
    >
      <span
        className="fa fa-search emoji-picker__search-icon"
      />
      <FormattedMessage
        defaultMessage="Search for an emoji"
        id="emoji_picker.search_emoji"
        values={Object {}}
      >
        <ForwardRef(LocalizedInput)
          className="emoji-picker__search"
          data-testid="emojiInputSearch"
          id="emojiPickerSearch"
          placeholder={
            Object {
              "defaultMessage": "Search Emoji",
              "id": "emoji_picker.search",
            }
          }
          type="text"
        >
          <input
            className="emoji-picker__search"
            data-testid="emojiInputSearch"
            id="emojiPickerSearch"
            placeholder="Search Emoji"
            type="text"
          />
        </ForwardRef(LocalizedInput)>
      </FormattedMessage>
    </div>
  </ForwardRef(EmojiPickerSearchInput)>
</IntlProvider>
`;
